src/app/components/forms/form/terra-form.component.ts

```diff
@@ -141,7 +145,7 @@ export class TerraFormComponent implements ControlValueAccessor, OnChanges, OnIn
             });
             return result;
         }
-        return isNullOrUndefined(field.defaultValue) ? null : field.defaultValue;
+        return isNullOrUndefined(field.defaultValue) ? null : this.cloneDefaultValue(field.defaultValue);
```

Comment:
Die beiden Methoden könnten eigentlich super in den TerraFormHelper verschoben werden, oder nicht?